ezxenstore.opam

```diff
@@ -6,19 +6,13 @@ homepage: "https://github.com/xapi-project/ezxenstore"
 bug-reports: "https://github.com/xapi-project/ezxenstore/issues"
 dev-repo: "https://github.com/xapi-project/ezxenstore.git"
 
-build: [
-  make	
-]
-install: [
-  make "install"
-]
-remove: [
-  "ocamlfind" "remove" "ezxenstore"
-]
+build: [[ "jbuilder" "build" "-p" name "-j" jobs ]]
+build-test: [[ "jbuilder" "runtest" "-p" name "-j" jobs ]]
```

Comment:
Maybe we should not add this, because these are not unit tests - this requires a running xenstored to work, therefore the tests fail on my machine.